src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  background-color: black;
+}
+
+.zovn {
+  background-color: #308dc0;
+  width: 328px;
+  border: 1px solid #e6635f;
+  padding: 10px;
+  margin: 20px 12px;
+}
+
+#seredn {
+  margin: 40px 12px;
+}
```

Comment:
1. don't style by id, it has higher specificity, use classes
2. class names should be valid English words